serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

```diff
@@ -138,7 +138,7 @@ For example using VSCode, there are the commands needed:
 
 [source,bash,subs="attributes+"]
 ----
-curl -S https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_sonataflow_devmode.yaml > workflow_devmode.yaml
+curl -S https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sonataflow.org_v1alpha08_sonataflow_devmodeWithConfigMapAndExternalResource.yaml > workflow_devmode.yaml
```

Comment:
```suggestion
curl -S https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sonataflow.org_v1alpha08_sonataflow_devmode.yaml > workflow_devmode.yaml
```

Reply:
@ricardozanini just FYI there is no such file, that was my main point. We should probably remove that ConfigMap, which is not even used in the workflow definition and rename it to `sonataflow.org_v1alpha08_sonataflow_devmode.yaml` as you proposed.

Reply:
Oh it's this one: https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/main/test/testdata/sonataflow.org_v1alpha08_sonataflow_devmode.yaml

Reply:
@ricardozanini sure, now it makes sense. Although, I think we should revisit the original CR as that config map is clearly not used there.